### PR TITLE
Unfortunately had to comment out some copy on the about map view since it...

### PR DESCRIPTION
**Pull request by mashcode**
... was over running the mobile box again

### CODE REVIEW

File: _layouts/gumby-ui.html [hunk 1]
```diff
@@ -167,9 +167,9 @@ <h1 class="inview txt" gumby-classname="onscreen" gumby-offset="5">The Story</h1
 <div data-target="about-map" class="space-large-screens"></div>
 <div class="about-box">
     <h1 class="inview txt" gumby-classname="onscreen" gumby-offset="5">The Map</h1>
-    <p class="inview txt" gumby-classname="onscreen" gumby-offset="5">This is a map of regulated day care centers within the five boroughs of New York City created from data scraped off the NYCDoHMH and NYSOFCS government web sites. This would not have been possible without the generous assistance of the</p>
+    <p class="inview txt" gumby-classname="onscreen" gumby-offset="5">This is a map of regulated day care centers within the five boroughs of New York City created from data scraped off the NYCDoHMH and NYSOFCS government web sites. This would not have been possible without the generous assistance from the</p>
     <p class="inview link" gumby-classname="onscreen" gumby-offset="5"><a href="http://betanyc.us/" target="_blank">BetaNYC civic hacker community</a>.</p>
-    <p class="inview txt" gumby-classname="onscreen" gumby-offset="5">The data was last updated June 12th 2014.</p>
+    <!--p class="inview txt" gumby-classname="onscreen" gumby-offset="5">The data was last updated June 12th 2014.</p-->
```
Comment (schmiani) [inline _layouts/gumby-ui.html:172]: It should say updated on July 14th everywhere now! I thought I updated it?

File: _layouts/nyccdb.html [hunk 2]
```diff
@@ -130,9 +130,9 @@ <h1 class="inview txt" gumby-classname="onscreen" gumby-offset="5">The Story</h1
 <div data-target="about-map" class="space-large-screens"></div>
 <div class="about-box">
     <h1 class="inview txt" gumby-classname="onscreen" gumby-offset="5">The Map</h1>
-    <p class="inview txt" gumby-classname="onscreen" gumby-offset="5">This is a map of regulated day care centers within the five boroughs of New York City created from data scraped off the NYCDoHMH and NYSOFCS government web sites. This would not have been possible without the generous assistance of the</p>
+    <p class="inview txt" gumby-classname="onscreen" gumby-offset="5">This is a map of regulated day care centers within the five boroughs of New York City created from data scraped off the NYCDoHMH and NYSOFCS government web sites. This would not have been possible without the generous assistance from the</p>
     <p class="inview link" gumby-classname="onscreen" gumby-offset="5"><a href="http://betanyc.us/" target="_blank">BetaNYC civic hacker community</a>.</p>
-    <p class="inview txt" gumby-classname="onscreen" gumby-offset="5">The data was last updated June 12th 2014.</p>
+    <!--p class="inview txt" gumby-classname="onscreen" gumby-offset="5">The data was last updated June 12th 2014.</p-->
```
Comment (schmiani) [inline _layouts/nyccdb.html:135]: Same thing here: I scraped July 14th, I think it's important for the demo tomorrow that we have up-to-date information

Comment (mashcode) [inline _layouts/nyccdb.html:135]: The copy over runs the window height for The Map. What I will do is force
it to scroll.

We could also put an update stamp in the footer

On Jul 18, 2014, at 5:23 PM, Anita Schmid notifications@github.com wrote:

In _layouts/nyccdb.html:

> ```
>  <p class="inview link" gumby-classname="onscreen" gumby-offset="5"><a href="http://betanyc.us/" target="_blank">BetaNYC civic hacker community</a>.</p>
> ```
> -    <p class="inview txt" gumby-classname="onscreen" gumby-offset="5">The data was last updated June 12th 2014.</p>
> -    

Same thing here: I scraped July 14th, I think it's important for the demo
tomorrow that we have up-to-date information

—
Reply to this email directly or view it on GitHub
https://github.com/childcaremap/NYCdaycare/pull/72/files#r15134401.

Comment (schmiani) [inline _layouts/nyccdb.html:135]: Or we just take out "within the 5 boroughs" for now? Or put the date in the footer that is a great idea!

**Pull request merged**